global.json

```diff
@@ -1,6 +1,6 @@
 {
-  "projects": [ "src" ],
-    "sdk": {
+  "projects": [ "src", "tests" ],
```

Comment:
Ummmmm.... I think it was because I was having versioning problems with xunit. This might be ok now.